mteb/models/misc_models.py

```diff
@@ -0,0 +1,1582 @@
+        "ara_Arab",
+        "asm_Beng",
+        "bam_Latn",
+        "ben_Beng",
+        "cat_Latn",
+        "eng_Latn",
+        "spa_Latn",
+        "eus_Latn",
+        "fon_Latn",
+        "fra_Latn",
+        "guj_Gujr",
+        "hin_Deva",
+        "ind_Latn",
+        "ibo_Latn",
+        "kik_Latn",
+        "kan_Knda",
+        "lug_Latn",
+        "lin_Latn",
+        "mal_Mlym",
+        "mar_Deva",
+        "nep_Deva",
+        "nso_Latn",
+        "nya_Latn",
+        "ori_Orya",
+        "pan_Guru",
+        "por_Latn",
+        "run_Latn",
+        "kin_Latn",
+        "sna_Latn",
+        "sot_Latn",
+        "swa_Latn",
+        "tam_Taml",
+        "tel_Telu",
+        "tsn_Latn",
+        "tso_Latn",
+        "tum_Latn",
+        "twi_Latn",
+        "urd_Arab",
+        "vie_Latn",
+        "wol_Latn",
+        "xho_Latn",
+        "yor_Latn",
+        "zho_Hans",
+        "zul_Latn",
+    ],
+    loader=None,
+    n_parameters=None,
+    memory_usage=None,
+    max_tokens=None,
+    embed_dim=None,
+    license="bigscience-bloom-rail-1.0",
+    open_weights=True,
+    public_training_data=False,
+    public_training_code=None,
+    framework=["PyTorch"],
+    reference="https://huggingface.co/izhx/udever-bloom-3b",
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
+    superseded_by=None,
+)
+izhx__udever_bloom_560m = ModelMeta(
+    name="izhx/udever-bloom-560m",
+    revision="b2a723e355946ec5a5c5fbed3459766627ded2bb",
+    release_date="2023-10-24",
+    languages=[
+        "aka_Latn",
+        "ara_Arab",
+        "asm_Beng",
+        "bam_Latn",
+        "ben_Beng",
+        "cat_Latn",
+        "eng_Latn",
+        "spa_Latn",
+        "eus_Latn",
+        "fon_Latn",
+        "fra_Latn",
+        "guj_Gujr",
+        "hin_Deva",
+        "ind_Latn",
+        "ibo_Latn",
+        "kik_Latn",
+        "kan_Knda",
+        "lug_Latn",
+        "lin_Latn",
+        "mal_Mlym",
+        "mar_Deva",
+        "nep_Deva",
+        "nso_Latn",
+        "nya_Latn",
+        "ori_Orya",
+        "pan_Guru",
+        "por_Latn",
+        "run_Latn",
+        "kin_Latn",
+        "sna_Latn",
+        "sot_Latn",
+        "swa_Latn",
+        "tam_Taml",
+        "tel_Telu",
+        "tsn_Latn",
+        "tso_Latn",
+        "tum_Latn",
+        "twi_Latn",
+        "urd_Arab",
+        "vie_Latn",
+        "wol_Latn",
+        "xho_Latn",
+        "yor_Latn",
+        "zho_Hans",
+        "zul_Latn",
+    ],
+    loader=None,
+    n_parameters=None,
+    memory_usage=None,
+    max_tokens=None,
+    embed_dim=None,
+    license="bigscience-bloom-rail-1.0",
+    open_weights=True,
+    public_training_data=False,
+    public_training_code=None,
+    framework=["PyTorch"],
+    reference="https://huggingface.co/izhx/udever-bloom-560m",
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
```

Comment:
For example, this is a finetune from `"bigscience/bloom-560m"` as stated on the model card. This can be found in `config.json`:
```
"_name_or_path": "bigscience/bloom-560m"
```